plugins/push/frontend/public/stylesheets/main.scss

```diff
@@ -25,4 +26,286 @@
 .blinker-wrapper {
     padding: 0 6px;
     border-radius: 4px;
+    font-size: typography.$text-medium-size
+}
+
+.mobile-preview {
```

Comment:
Don't want to be a nuisance but "mobile-preview" sounds too generic, it may pollute the global scope. I have no idea about the context though. What would you say @itsiprikshit ?

Reply:
I will add the `cly-vue-push-notification` prefix since it's being used by push plugin anyways.

Reply:
yep sounds good @LumbardhAgaj.

Reply:
Ideally every block should have cly-vue- prepended to them.